modules/oadp-configuring-noobaa-for-dr.adoc

```diff
@@ -4,15 +4,17 @@
 
 :_content-type: PROCEDURE
 [id="oadp-configuring-noobaa-for-dr_{context}"]
-= Configuring NooBaa for disaster recovery on {rh-storage}
+= Configuring Multicloud Object Gateway for disaster recovery on {rh-storage}
```

Comment:
@RichardHoch 
I would add the acronym here: "...Multicloud Object Gateway (MCG) for...", otherwise the next sentence becomes impossibly heavy.